### PR TITLE
:pencil: update faq.md about vue-router error

### PR DESCRIPTION
**Pull request by eynol**
##### Checklist

- [x] `npm test` passes
- [ ] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

add error explain and fix options about `vue-router`

### CODE REVIEW

File: docs/zh/faq/README.md [hunk 1]
```diff
@@ -49,6 +49,23 @@ module.exports = {
 };
 ```
 
+## Vue Router 报错 `Uncaught TypeError: Cannot redefine property: $router`
+
+qiankun中的代码使用Proxy去代理父页面的window，来实现的沙箱，在子应用中访问`window.Vue`时，会先在自己的window里查找有没有`Vue`属性，如果没有就去父应用里查找。
```
Comment (howel52) [inline docs/zh/faq/README.md:54]: 中英混排的话 是不是可以加点空格

Comment (eynol) [reply docs/zh/faq/README.md:54]: 有道理～

**Comment by howel52** [comment]
LGTM

File: docs/zh/faq/README.md [hunk 2]
```diff
@@ -275,4 +292,4 @@ const render = ($) => {
 
 你也可以直接参照 examples 中 purehtml 部分的[代码](https://github.com/umijs/qiankun/tree/master/examples/purehtml)
 
-同时，你也需要开启相关资源的 CORS，具体请参照[此处](/zh/faq/#%E5%AD%90%E5%BA%94%E7%94%A8%E9%9D%99%E6%80%81%E8%B5%84%E6%BA%90%E4%B8%80%E5%AE%9A%E8%A6%81%E6%94%AF%E6%8C%81%E8%B7%A8%E5%9F%9F%E5%90%97%EF%BC%9F)
\ No newline at end of file
+同时，你也需要开启相关资源的 CORS，具体请参照[此处](/zh/faq/#%E5%AD%90%E5%BA%94%E7%94%A8%E9%9D%99%E6%80%81%E8%B5%84%E6%BA%90%E4%B8%80%E5%AE%9A%E8%A6%81%E6%94%AF%E6%8C%81%E8%B7%A8%E5%9F%9F%E5%90%97%EF%BC%9F)
```
Comment (kuitos) [inline docs/zh/faq/README.md:295]: 这里是改了啥？

**Pull request merged**